services/klatchat_observer/controller.py

```diff
@@ -509,9 +511,7 @@ def handle_message(self, data: dict):
                 or Recipients.UNRESOLVED
             )
             handler_method = self.recipient_to_handler_method.get(recipient)
-            if not handler_method:
-                LOG.warning(f"Failed to get handler message for recipient={recipient}")
```

Comment:
Is this log worth keeping? Without it, I'm not sure there's any way to see in logs if a message was received

Reply:
> Is this log worth keeping? Without it, I'm not sure there's any way to see in logs if a message was received

It was pointlessly spamming on every single message that occurred in klat, so I believe we can remove it for now, as we did not really use it so far